site/docs/relations/logical_relations.md

```diff
@@ -51,12 +51,15 @@ Read definition types (like the rest of the features in Substrait) are built by
 #### Virtual Table
 
 A virtual table is a table whose contents are embedded in the plan itself.  The table data
-is encoded as records consisting of literal values.
+is encoded as records consisting of literal values or an expression that can be resolved without referencing any input data.
```

Comment:
minor/nit
```
literal values or an expression that can
  to
literal values or expressions that can
```
it's a little dramatically wonky that expression is not pluralized.

Reply:
fixed it